src/components/NewHathorButton.js

```diff
@@ -77,6 +77,7 @@ const style = StyleSheet.create({
     borderRadius: 8,
     backgroundColor: COLORS.textColor,
     alignSelf: 'stretch',
+    padding: 16,
```

Comment:
Oh! It should not cause any trouble for any button that we have because these buttons are large enough. However, I will pass trough the views to double check.

Reply:
It didn't changed any button appearance nor has caused any issue so far. We added this style because the button title "Register Nano Contract" a bit longer than usual and can touches the button border, which justifies the padding.